src/main/java/com/aws/greengrass/mqttbridge/clients/MQTTClient.java

```diff
@@ -281,6 +277,7 @@ private void reconnectAndResubscribe() {
                 try {
                     Thread.sleep(waitBeforeRetry * 1000);
                 } catch (InterruptedException er) {
+                    Thread.currentThread().interrupt();
                     LOGGER.atError().setCause(er).log("Failed to reconnect");
```

Comment:
Customers get scared by error logs. I don't think this is actually an error condition - getting interrupted likely means we're shutting down, right? Maybe we can downgrade to info and say we're aborting due to interruption rather than "Failed to reconnect"